modules/service/src/main/scala/lucuma/odb/service/GuideService.scala

```diff
@@ -276,8 +278,8 @@ object GuideService {
         (Site.GN, AgsParams.GmosAgsParams(mode.fpu.asLeft.some, PortDisposition.Side), mode.centralWavelength)
       case mode: gmos.longslit.Config.GmosSouth =>
         (Site.GS, AgsParams.GmosAgsParams(mode.fpu.asRight.some, PortDisposition.Side), mode.centralWavelength)
-      case mode: flamingos2.longslit.Config =>
-        ???
+      case mode: flamingos2.longslit.Config     =>
+        (Site.GS, AgsParams.Flamingos2AgsParams(Flamingos2LyotWheel.F16, Flamingos2FpuMask.Builtin(mode.fpu), PortDisposition.Side), mode.filter.wavelength)
```

Comment:
Are these values a temprary measure?

Reply:
we always use F16 lyot, the other options are for Gmes + F2 which we never use

Reply:
Eventually we neeed to parametrize the port